README.md

```diff
@@ -1,3 +1,7 @@
 # Apprentice Page
 
-Sparkbox's new apprentice page developed by the current apprentices.
\ No newline at end of file
+Sparkbox's new apprentice page developed by the current apprentices.
+
+## Development
+
+We really need to tell people to use node for this project.
```

Comment:
Hey! Great that you suggest we should do this. How about actually spelling that out?